File: model/src/main/java/net/artux/pda/model/map/Point.java
package net.artux.pda.model.map;

import java.io.Serializable;
import java.util.HashMap;
import java.util.List;

import lombok.Data;

@Data
public class Point implements Serializable {

    private int type;
    private String name;
    private String pos;
    private HashMap<String, String> data;
    private HashMap<String, List<String>> condition;


}

File: model/src/main/java/net/artux/pda/model/map/Point.kt
package net.artux.pda.model.map

import java.io.Serializable
import java.util.UUID

data class Point(
    var id: UUID,
    var type: Int = 0,
    var name: String = "point",
    var pos: String = "500:500",
) : Serializable {
    var data: HashMap<String, String> = HashMap()
    var condition: HashMap<String, List<String>> = HashMap()
}